general-information.md

```diff
@@ -177,6 +178,25 @@ Note if implementing the beta feature [Geography Driven Events](/general-informa
 
 [Top][toc]
 
+## Trips
+
+Counting trips is a common analysis with MDS data, but in some instances when
+a user rents a vehicle the rental may end up being very short because the
+vehicle is unusable or because the customer changes their mind.
+To facilitate a common definition of trips for analysis, MDS defines a trip
+as a user taking control of a vehicle for 60 seconds or more.
+If a user reserves a vehicle and cancels before taking control of the vehicle
+that reservation does not count as a trip, nor does a user taking control of
+the vehicle and canceling or ending the trip in under 60 seconds.
+
+Providers are still expected to report these events in MDS endpoints,
+but parties may use this definition as a shared reference at the
+recommendation of the MDS community.
+
+For reference on the development of this definition please refer to
```

Comment:
I think you can cut the link to this Issue. Instead we may add a new Issue or Discussion area that can be referenced here for people to leave feedback.